sample-queries/sample-queries.json

```diff
@@ -73,6 +73,17 @@
         "docLink": "https://docs.microsoft.com/en-us/graph/api/todo-list-lists?view=graph-rest-beta",
         "skipTest": false
     },
+    {
+        "id": "0FB601F7-2170-40DE-88B7-4D7387C7B547",
+        "category": "Users",
+        "method": "GET",
+        "humanName": "count the guest users in your tenant (raw count)",
+        "requestUrl": "/v1.0/users/$count?$filter=userType eq 'guest'",
+        "headers": [ { "name": "ConsistencyLevel", "value": "eventual" } ],
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/user-list?view=graph-rest-beta&tabs=http#example-6-get-only-a-count-of-users",
+        "tip": "You are using the advanced query capabilities for Directory Objects, please send us feedback here: https://aka.ms/aadmgs",
+        "skipTest": false
+    },
     {
```

Comment:
@Licantrop0 this Sample query seems to have two ids and two categories.

Reply:
where?